src/shared/components/home/site-form.tsx

```diff
@@ -687,6 +689,7 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
             type="button"
             className="btn btn-sm bg-success ml-2"
             onClick={linkEvent(key, this.handleAddInstance)}
+            style={"width: 2rem; height: 2rem;"}
```

Comment:
There's got to be some bootstrap classes for this.

Reply:
Unfortunately, [not out of the box](https://getbootstrap.com/docs/5.2/utilities/sizing/#relative-to-the-parent). However, [utility classes can be generated](https://getbootstrap.com/docs/5.2/utilities/api/#using-the-api).